<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Leighton Pritchard">

<title>2&nbsp; PET hydrolase homolog cloning – BM923 Synthetic Biology Workshop</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./pkfA2RED.html" rel="next">
<link href="./adpA.html" rel="prev">
<link href="./icon_32.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-72c93205fb41c15e2e398b4f9a505ee2.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./pET.html"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">PET hydrolase homolog cloning</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">BM923 Synthetic Biology Workshop</a> 
        <div class="sidebar-tools-main">
    <a href="https://github.com/mafeeney/BM923synbio" title="Source Code" class="quarto-navigation-tool px-1" aria-label="Source Code"><i class="bi bi-github"></i></a>
  <a href="" class="quarto-reader-toggle quarto-navigation-tool px-1" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./adpA.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Cloning AdpA</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./pET.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">PET hydrolase homolog cloning</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./pkfA2RED.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Deleting pkfA2 (lambda Red-based recombineering)</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./pkfA2Crispr.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Deleting pkfA2 (using CRISPR)</span></span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#cloning-pet-hydrolase-homologs-for-heterologous-expression-in-pseudomonas-gibson-assembly" id="toc-cloning-pet-hydrolase-homologs-for-heterologous-expression-in-pseudomonas-gibson-assembly" class="nav-link active" data-scroll-target="#cloning-pet-hydrolase-homologs-for-heterologous-expression-in-pseudomonas-gibson-assembly"><span class="header-section-number">2.1</span> Cloning PET hydrolase homologs for heterologous expression in Pseudomonas (Gibson assembly)</a>
  <ul class="collapse">
  <li><a href="#rationale-for-design-choices" id="toc-rationale-for-design-choices" class="nav-link" data-scroll-target="#rationale-for-design-choices"><span class="header-section-number">2.1.1</span> Rationale for design choices</a></li>
  <li><a href="#obtaining-the-sequence-for-the-pet-hydrolase" id="toc-obtaining-the-sequence-for-the-pet-hydrolase" class="nav-link" data-scroll-target="#obtaining-the-sequence-for-the-pet-hydrolase"><span class="header-section-number">2.1.2</span> Obtaining the sequence for the PET hydrolase</a></li>
  <li><a href="#obtaining-the-sequence-for-the-plasmid" id="toc-obtaining-the-sequence-for-the-plasmid" class="nav-link" data-scroll-target="#obtaining-the-sequence-for-the-plasmid"><span class="header-section-number">2.1.3</span> Obtaining the sequence for the plasmid</a></li>
  <li><a href="#in-silico-design-of-your-gibson-assembly" id="toc-in-silico-design-of-your-gibson-assembly" class="nav-link" data-scroll-target="#in-silico-design-of-your-gibson-assembly"><span class="header-section-number">2.1.4</span> In silico design of your Gibson Assembly</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><ul><li><a href="https://github.com/mafeeney/BM923synbio/edit/main/pET.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/mafeeney/BM923synbio/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">PET hydrolase homolog cloning</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="cloning-pet-hydrolase-homologs-for-heterologous-expression-in-pseudomonas-gibson-assembly" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="cloning-pet-hydrolase-homologs-for-heterologous-expression-in-pseudomonas-gibson-assembly"><span class="header-section-number">2.1</span> Cloning PET hydrolase homologs for heterologous expression in Pseudomonas (Gibson assembly)</h2>
<p>In this example we will clone the pET hydrolase from <em>Ideonella sakaiensis</em> into an expression vector (pMMPc) with a strong constitutive promoter for expression in <em>Pseudomonas putida</em>, using Gibson assembly.</p>
<section id="rationale-for-design-choices" class="level3" data-number="2.1.1">
<h3 data-number="2.1.1" class="anchored" data-anchor-id="rationale-for-design-choices"><span class="header-section-number">2.1.1</span> Rationale for design choices</h3>
<p>Strain choice: <em>Pseudomonas putida</em> is tolerant to solvents, has a robust metabolism and capacity to withstand stresses, and has been used as a chassis organism for industrial biotechnology purposes previously, making it an ideal choice for our PET hydrolase expression system. (c.f. Weimer et al 2020 - https://doi.org/10.1007/s00253-020-10811-9)</p>
<p>Vector choice: Need a plasmid that will replicate in Pseudomonas, with a strong constitutive promoter. Xu et al (2013) created some appropriate vectors, c.f. doi: 10.1128/AEM.03746-12 - we will choose pMMPc (Genbank accession KC544266).</p>
<p>Choice of PET hydrolase: <em>Ideonella sakaiensis</em> has been shown to degrade PET (and can use it as a sole carbon/energy source) doi:10.1126/science.aad6359. We will begin by cloning the well-characterized PET hydrolase from this organism (further optimisation of the hydrolase properties using gene engineering, or exploration of potentially more active/useful homologs may be necessary in the future).</p>
</section>
<section id="obtaining-the-sequence-for-the-pet-hydrolase" class="level3" data-number="2.1.2">
<h3 data-number="2.1.2" class="anchored" data-anchor-id="obtaining-the-sequence-for-the-pet-hydrolase"><span class="header-section-number">2.1.2</span> Obtaining the sequence for the PET hydrolase</h3>
<ol type="1">
<li>Search the UniProt database for “PET hydrolase” (<a href="#fig-pet1" class="quarto-xref">Figure&nbsp;<span>2.1</span></a>).</li>
</ol>
<div id="fig-pet1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet1.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.1: Searching the UniProt database
</figcaption>
</figure>
</div>
<ol start="2" type="1">
<li>Click on the search result that you want (in our case, the PET hydrolase from <em>Ideonella sakaiensis</em>/<em>Piscinibacter sakaiensis</em>). This should take you to the page for this protein, with information about its function, etc (<a href="#fig-pet2" class="quarto-xref">Figure&nbsp;<span>2.2</span></a>).</li>
</ol>
<div id="fig-pet2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet2.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.2: The I. sakaiensis PET Hydrolase entry in UniProt
</figcaption>
</figure>
</div>
<ol start="3" type="1">
<li>The uniprot accession # for this PET hydrolase is A0A0K8P6T7, which you will need to import the data into Benchling.</li>
</ol>
<p>In Benchling, you may wish to start a new project (You can do this by clicking on the + icon in the left-hand toolbar, selecting “Project”, and then giving your new project an informative name and description, then clicking “Create project”.)</p>
<p>To import the sequence of the PET hydrolase, click on the + icon, then click AA sequence &gt; Import AA sequence (<a href="#fig-pet3" class="quarto-xref">Figure&nbsp;<span>2.3</span></a>)</p>
<div id="fig-pet3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet3.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.3: Importing an AA sequence into Benchling
</figcaption>
</figure>
</div>
<p>Click “Import from database”, specify the Accession number (A0A0K8P6T7), Database (UniProt), and Project folder (the folder you just created), then click “Import” (<a href="#fig-pet4" class="quarto-xref">Figure&nbsp;<span>2.4</span></a>).</p>
<div id="fig-pet4" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet4.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.4: Specifying the Uniprot accession to import
</figcaption>
</figure>
</div>
<p>You should then be able to open the imported sequence and observe the amino acid sequence and its features (<a href="#fig-pet5" class="quarto-xref">Figure&nbsp;<span>2.5</span></a>).</p>
<div id="fig-pet5" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet5.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.5: The PET hydrolase sequence imported into Benchling
</figcaption>
</figure>
</div>
<ol start="4" type="1">
<li>You will now need to create a reverse translation of the amino acid sequence to derive the coding sequence necessary for its production.</li>
</ol>
<p>Select the entire sequence, right click on it, and then then click on “Back translate”.</p>
<p>Specify the options to optimize your back translation: you wish to codon optimize for expression in <em>Pseudomonas putida</em>, and you should add EcoRI as a cut site to avoid (we will be using this site to cut the plasmid for our Gibson assembly reaction.) (<a href="#fig-pet6" class="quarto-xref">Figure&nbsp;<span>2.6</span></a>)</p>
<div id="fig-pet6" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet6.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.6: Specifying options to reverse translate/back translate the PET hydrolase AA sequence
</figcaption>
</figure>
</div>
<p>Click “Preview optimization” when you are happy with the settings, and then “Save as new sequence”, making sure to save it in the correct Project folder.</p>
<ol start="6" type="1">
<li>Quality check and modify the sequence as necessary:</li>
</ol>
<p>A. Select the entirety of the sequence, then click Create &gt; Translation &gt; Forward to examine the translation of your new nucleotide sequence (<a href="#fig-pet7" class="quarto-xref">Figure&nbsp;<span>2.7</span></a>).</p>
<div id="fig-pet7" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet7.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.7: The PET hydrolase sequence in Benchling, with translation added
</figcaption>
</figure>
</div>
<p>In this case, note that while there is a start codon present, there is no stop codon - we need to add one.</p>
<p>Right click at the end of the sequence, then select “Insert bases” and add a stop codon (any of the 3 stop codons is fine).</p>
<div id="fig-pet8" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet8.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.8: Adding a stop codon to the sequence
</figcaption>
</figure>
</div>
<p>B. You may also wish to add an RBS at the beginning of the sequence (it doesn’t look like there is one present in the vector that we’re using). You can do this using the same method (right click, “Insert bases”, add the desired bases. You could use for example RBS 33 from Aparicio et al 2020, which has been shown to give high expression in <em>P. putida</em> (https://doi.org/10.1016/j.isci.2020.100946) - <u>TAAGGAGGT</u>AAAAACAT<b>ATG</b> (RBS underlined, start in bold))</p>
<div id="fig-pet9" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet9-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet9.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet9-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.9: Adding an RBS to the sequence
</figcaption>
</figure>
</div>
<p>C. You can double-check that this sequence is correct by confirming (e.g., using BLASTx) that it matches the <em>I. sakaiensis</em> PET hydrolase protein.</p>
<p>This is the sequence that you would then have synthesized (e.g., by GenScript or another DNA synthesis company) for use in your Gibson assembly reaction.</p>
</section>
<section id="obtaining-the-sequence-for-the-plasmid" class="level3" data-number="2.1.3">
<h3 data-number="2.1.3" class="anchored" data-anchor-id="obtaining-the-sequence-for-the-plasmid"><span class="header-section-number">2.1.3</span> Obtaining the sequence for the plasmid</h3>
<p>The NCBI accession number for the plasmid we wish to use is KC544266. You can import this into Benchling by clicking the + icon, then clicking DNA/RNA sequence &gt; Import DNA RNA sequences.</p>
<div id="fig-pet10" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet10-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet10.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet10-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.10: Importing a DNA sequence into Benchling
</figcaption>
</figure>
</div>
<p>In the pop-up window, click “Import from database” and paste or type the accession number into the “Sequence” box, then click “Search”, and then “Import” (<a href="#fig-pet11" class="quarto-xref">Figure&nbsp;<span>2.11</span></a>).</p>
<div id="fig-pet11" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet11-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet11.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet11-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.11: Specifying an NCBI accession number to import
</figcaption>
</figure>
</div>
<p>This should give you the plasmid sequence for the plasmid (<a href="#fig-pet12" class="quarto-xref">Figure&nbsp;<span>2.12</span></a>). Note the constitutive promoter (Pc) that we want to use for our cloning project. There is a multiple cloning site (MCS) just downstream of it, which contains EcoRI, BclI, AflIII etc. sites … we have chosen above to use the EcoRI site (but note that we could have chosen any of the sites in the MCS).</p>
<div id="fig-pet12" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet12-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet12.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet12-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.12: Plasmid map of the vector imported into Benchling
</figcaption>
</figure>
</div>
<p>You can confirm the presence of the EcoRI site manually as well (click on the scissors icon in the right-hand menu and then search for “EcoRI” (<a href="#fig-pet13" class="quarto-xref">Figure&nbsp;<span>2.13</span></a>).</p>
<div id="fig-pet13" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet13-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet13.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet13-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.13: Confirming the EcoRI site in your imported vector sequence
</figcaption>
</figure>
</div>
</section>
<section id="in-silico-design-of-your-gibson-assembly" class="level3" data-number="2.1.4">
<h3 data-number="2.1.4" class="anchored" data-anchor-id="in-silico-design-of-your-gibson-assembly"><span class="header-section-number">2.1.4</span> In silico design of your Gibson Assembly</h3>
<ol type="1">
<li>Click on “Assembly” in the bottom right hand corner of your screen &gt; “Assembly Wizard”, and then in the pop-up window Pick Assembly Strategy, select “Gibson” (<a href="#fig-pet14" class="quarto-xref">Figure&nbsp;<span>2.14</span></a>).</li>
</ol>
<div id="fig-pet14" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet14-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet14.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet14-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.14: Choosing an Assembly Strategy
</figcaption>
</figure>
</div>
<p>This should bring up a message encouraging you to try the new combinatorial assembly tool - click that link.</p>
<ol start="2" type="1">
<li>In the pop-up menu, add the name of your plasmid that you are aiming to create (Tip: choose something informative, e.g.&nbsp;pMMPc-PET), and be sure to specify the correct project folder.</li>
</ol>
<p>Check the other specifications: you are assembling two fragments, to create a circular construct, and you will be producing fragments by “Digest with restriction enzyme”. The defualts for min/max length and min Tm of your binding regions are fine.</p>
<p>When you are happy with the specifications, click “Save” (<a href="#fig-pet15" class="quarto-xref">Figure&nbsp;<span>2.15</span></a>)</p>
<div id="fig-pet15" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet15-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet15.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet15-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.15: Specifying DNA assembly parameters
</figcaption>
</figure>
</div>
<p>You should now get an assembly tool that allows you to specify the fragments you will use (<a href="#fig-pet16" class="quarto-xref">Figure&nbsp;<span>2.16</span></a>).</p>
<div id="fig-pet16" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet16-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet16.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet16-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.16: Specifying the DNA fragments to use in your assembly
</figcaption>
</figure>
</div>
<p>For BIN 1 (Backbone), click the + symbol &gt; Open sequences &gt; KC544266 (<a href="#fig-pet17" class="quarto-xref">Figure&nbsp;<span>2.17</span></a>)</p>
<div id="fig-pet17" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet17-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet17.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet17-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.17: Adding the vector backbone
</figcaption>
</figure>
</div>
<p>Specify the 5’ enzyme and 3’ enzyme you will use (EcoRI) and the EcoRI site on the map as the Start and End sites (by clicking on View: Plasmid map to see the EcoRI site if necessary) (<a href="#fig-pet18" class="quarto-xref">Figure&nbsp;<span>2.18</span></a>), then click “Add”.</p>
<div id="fig-pet18" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet18-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet18.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet18-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.18: Specifying the vector start and end and enzyme details
</figcaption>
</figure>
</div>
<p>For BIN 2 (Insert 1), change the method to “Add new overlaps using PCR” and then click the + sign to add your fragment &gt; Open sequences &gt; the name of your codon optimized sequence (<a href="#fig-pet19" class="quarto-xref">Figure&nbsp;<span>2.19</span></a>).</p>
<div id="fig-pet19" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet19-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet19.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet19-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.19: Adding the insert to your assembly
</figcaption>
</figure>
</div>
<p>This should bring up your correct fragment to insert (<a href="#fig-pet20" class="quarto-xref">Figure&nbsp;<span>2.20</span></a>). Click “add”.</p>
<div id="fig-pet20" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet20-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet20.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet20-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.20: Adding the insert fragment
</figcaption>
</figure>
</div>
<p>Click “Autopopulate” in the “Constructs” box (<a href="#fig-pet21" class="quarto-xref">Figure&nbsp;<span>2.21</span></a>), and then “Assemble” in the top right hand corner”.</p>
<div id="fig-pet21" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet21-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet21.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet21-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.21: Autopopulate the constructs in your assembly
</figcaption>
</figure>
</div>
<p>Work through the Assemble DNA menu, specifying the folder where you would like to save your constructs, fragments, and primers (<span class="quarto-unresolved-ref">?fig-pet22</span>).</p>
<p><img src="0images/pet22.png" id="fig-pet22" class="img-fluid" alt="Specifying the locations to save your constructs, fragments, and primers"> Examine your final construct (<a href="#fig-pet23" class="quarto-xref">Figure&nbsp;<span>2.22</span></a>) to verify the construct has placed your PET hydrolase gene in the correct orientation relative to the Pc promoter, and to see the sequences of the primers that you would need to order to carry out this Gibson assembly (<a href="#fig-pet24" class="quarto-xref">Figure&nbsp;<span>2.23</span></a>).</p>
<div id="fig-pet23" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet23-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet23.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet23-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.22: Viewing your final construct
</figcaption>
</figure>
</div>
<div id="fig-pet24" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pet24-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="0images/pet24.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pet24-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.23: Viewing the primers needed for your assembly
</figcaption>
</figure>
</div>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./adpA.html" class="pagination-link" aria-label="Cloning AdpA">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Cloning AdpA</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./pkfA2RED.html" class="pagination-link" aria-label="Deleting pkfA2 (lambda Red-based recombineering)">
        <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Deleting pkfA2 (lambda Red-based recombineering)</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>BM923 Synthetic Biology Workshop - Dr.&nbsp;Morgan Feeney</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    <div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/mafeeney/BM923synbio/edit/main/pET.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/mafeeney/BM923synbio/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div>
    <div class="nav-footer-right">
<p>This book was built with <a href="https://quarto.org/">Quarto</a>.</p>
</div>
  </div>
</footer>




</body></html>